react-native-fitness.podspec

```diff
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
   s.authors      = package['contributors'].flat_map { |author| { author['name'] => author['email'] } }
   s.homepage     = package['homepage']
   s.license      = package['license']
-  s.platform     = :ios, "7.0"
+  s.platform     = :ios, "8.0"
```

Comment:
@Francesco-Voto Perhaps I should change that to 10.0? You bumped the versions in the project files here:
https://github.com/OvalMoney/react-native-fitness/pull/33

Reply:
Yes, it should be great!